source/image-handler/semantic-mapper.ts

```diff
@@ -42,19 +42,22 @@ export class SemanticMapper {
     w?: string | number
   }): ImageEdits {
 
-    const [width, height] = [queryParams?.w, queryParams?.h].map((dim) => {
+    if (this.isSvg || (!queryParams?.w && !queryParams?.h)) {
+      return this.EMPTY_IMAGE_EDITS;
+    }
+
+    const [width, height] = [queryParams.w, queryParams.h].map((dim) => {
       const intDim = parseInt(dim as string);
-      return isNaN(intDim) ? 0 : intDim;
+      return isNaN(intDim) ? null : intDim;
     });
 
     const resizeEdit: ImageEdits = { resize: {} };
-
-    // If width or height is 0 or missing, fit would be inside.
-    if (width === 0 || height === 0) {
+    // If width or height is null or missing, fit would be inside.
+    if (width === null || height === null) {
       resizeEdit.resize.fit = ImageFitTypes.INSIDE;
     }
-    resizeEdit.resize.width = width === 0 ? null : width;
-    resizeEdit.resize.height = height === 0 ? null : height;
+    resizeEdit.resize.width = width;
+    resizeEdit.resize.height = height;
```

Comment:
This change causes that If we set it to `w=0` we get a ServerError 500
![image](https://github.com/Livit/Portal.Cms.ImageHandler/assets/87857953/f79bb9be-b363-4f12-aad1-9a9de63a5801)

Likely `w=0` should be treated same as null.

Reply:
these runtime errors 🤦🏼 .